README.md

```diff
@@ -1,12 +1,105 @@
+          .route(routes ->
+                      // The server will respond only on POST requests
+                      // where the path starts with /test and then there is path parameter
+                  routes.post("/test/{param}", (request, response) ->
+                          response.sendString(request.receive()
+                                                     .asString()
+                                                     .map(s -> s + ' ' + request.param("param") + '!')
+                                                     .log("http-server"))))
+          .bindNow(); // Starts the server in a blocking fashion, and waits for it to finish its initialization
+```
+
+```java
+HttpClient.create()             // Prepares an HTTP client ready for configuration
+          .port(server.port())  // Obtains the server's port and provides it as a port to which this
+                                // client should connect
+          .post()               // Specifies that POST method will be used
+          .uri("/test/World")   // Specifies the path
+          .send(ByteBufFlux.fromString(Flux.just("Hello")))  // Sends the request body
+          .responseContent()    // Receives the response body
+          .aggregate()
+          .asString()
+          .log("http-client")
+          .block();
+
+```
+
+## Getting help
+Having trouble with `Reactor Netty`? We'd like to help!
+* If you are upgrading, read the [release notes](https://github.com/reactor/reactor-netty/releases)
+  for upgrade instructions and *new and noteworthy* features.
+* Ask a question - we monitor [Gitter](https://gitter.im/reactor/reactor) for questions.
+* Report bugs with `Reactor Netty` at [github.com/reactor/reactor-netty/issues](https://github.com/reactor/reactor-netty/issues).
+
+## Reporting Issues
+`Reactor Netty` uses `GitHub’s` integrated issue tracking system to record bugs and feature requests.
+If you want to raise an issue, please follow the recommendations below:
+* Before you log a bug, please [search the issue tracker](https://github.com/reactor/reactor-netty/search?type=Issues)
+  to see if someone has already reported the problem.
+* If the issue doesn't already exist, [create a new issue](https://github.com/reactor/reactor-netty/issues/new).
+* Please provide as much information as possible with the issue report, we like to know
+  the version of `Reactor Netty` that you are using, as well as your `Operating System` and
+  `JVM` version.
+
+## Building from Source
+You don't need to build from source to use `Reactor Netty` (binaries in
+[repo.spring.io](https://repo.spring.io)), but if you want to try out the latest and
+greatest, `Reactor Netty` can be easily built with the
+[gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). You also need JDK 1.8.
+
+```
+$ git clone https://github.com/reactor/reactor-netty.git
+$ cd reactor-netty
+$ ./gradlew build
```

Comment:
Do we have `publishToMavenLocal` task? Maybe we could mention it here instead of `build`, because `build` does not install it automatically

Reply:
yep ok it is just `./gradlew install`